friendships.go

```diff
@@ -0,0 +1,241 @@
+package twitter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+// NoRetweetIDs calls the Twitter /friendships/no_retweets/ids.json endpoint.
+func (c *Client) NoRetweetIDs(ctx context.Context, stringifyIDs bool) (*UserIDsResponse, error) {
+	values := url.Values{}
+	values.Set("stringify_ids", strconv.FormatBool(stringifyIDs))
```

Comment:
If we do that, I feel like we're making decisions for the SDK user which is usually not a great thing when writing an SDK but more acceptable when writing a Framework.